### ISSUE TITLE
Consider using Gitlocalize for localization

### ISSUE THREAD

**Issue by ilyaspiridonov** [ppy/osu-wiki#7910]
Translations are currently submitted via GitHub, which may make it difficult to track changes in the original files and update translations. 
Consider setting up a [Gitlocalize.com](https://gitlocalize.com) project (and sync existing translations)

**Comment by TicClick** [comment]
last time I checked, they didn't have support for file structure used on the wiki, and the freebie quota wasn't enough to fetch the whole repository anyway

**Comment by ilyaspiridonov** [comment]
@TicClick Yep, I've just experimented with the current file structure and it looks like it's not supported as is, indeed. Let me get back with this to our dev team to see if there's anything we can do. 
For the freebie quota we would be wiling to raise it a bit )

**Comment by TicClick** [comment]
> @TicClick Yep, I've just experimented with the current file structure and it looks like it's not supported as is, indeed. Let me get back with this to our dev team to see if there's anything we can do. For the freebie quota we would be wiling to raise it a bit )

could you do that for my account (handle's the same as on GitHub) so that I could toy with it once I get a second opportunity and see how it would fare for a project like ours, provided it had a different layout?

**Comment by TicClick** [comment]
I have checked this out a couple weeks ago by using https://osu.ppy.sh/wiki/en/Ranking_Criteria as a test subject, and the result is not pretty

the tooling itself is nice: once you set the repository up (I added a single file override though), you get access to an editor working in three modes:
- linking: you match pieces from the original and translation. it's automated *for the most part*, since it picks up Markdown features like headings, links, formatting, but if you accidentally mismatch two sections or items, the tool goes awry and thinks there's an off-by-one error, and the whole translation lags by a single item (or more if you felt brave and made more than one mistake), and you can't convince it otherwise.
- editing: you can edit a translation with corresponding blocks from the original being highlighted and replaced by generic tags: `[test](/wiki/Abc)` is `{a0}test{/a0}` now, etc
- proofreading: you can approve stuff, leave comments, etc

I didn't get to the actual PR submission part because I got fed up too quickly. the platform is good and has features like translation memory and stuff, semantic matching and whatnot, but it's too complicated for our purposes. we'll have to
- manualy process hundreds of articles to mark them up and realign with existing translations
- discard our front matter support, because apparently possible translation differences (such as extra article tags or YAML keys) are not accounted for
- set up a different review process
- get any contributor to learn the system
- discard our CI checks or make them less discoverable

all that versus plain text editing and a set of automated scripts we use for CI. granted, the thing is good, but again, you have to have an already assembled team of translators who know their ways around the platform, and maybe someone willing to STILL edit `.md` files by hand for things like cross-linking, etc, etc.

tl;dr: too complicated, potentially buggy, raises contribution entry threshold by A LOT. feel free to reopen if any of you are willing to experiment and get different experience
